src/stories/epics/EpicTable.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+import { EpicTable } from './EpicTable.stories';
+
+<Meta title="Epics/Table/Intro" component={EpicTable} />
+
+## Overview
+
+**Epics** are bodies of work that are large enough to be broken down to a number
+of smaller **tasks** within a given **project**.
+
+The **EpicTable** renders an [SLDS
+DataTable](https://react.lightningdesignsystem.com/components/data-tables/)
+accepting a collection of `epics` to render in the table. This allows users to
+view a list of epics and their related information created within a given
+project.
+
```

Comment:
I'm wondering if it helps to an image/rendering of the component on the docs page as well?